test/extension/utils.test.js

```diff
@@ -224,9 +224,10 @@ describe('Test extension utils', () => {
     })).to.be.true;
   });
 
-  it('deleteProject', async () => {
+  it('deleteProject', async (done) => {
     const spy = sandbox.spy(window.chrome.storage.sync, 'set');
     const deleted = await new Promise((resolve) => {
+      done();
```

Comment:
but this makes no sense. you preemptively finish the test.
`asyn` tests never need the `done`.

Reply:
Yes, that was nonsense. I reverted it and fixed the text framework properly in 5311af76a1b52d8352bed959cc164ab32a64b12b